health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/FacilityPopup.js

```
@@ -65,7 +66,27 @@ const FacilityPopUp = ({ details, onClose, updateDetails }) => {
+        businessIds: microplanId,
+   },
+    config: {
+        enabled: true,
+        select: (data) => {
+          return data?.ProcessInstances;
+     },
+    },
+  });
 
+  useEffect(() => {
+    if (processData && processData.some((instance) => instance.action === "FINALIZE_CATCHMENT_MAPPING")) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)